### PR TITLE
Folder restructure, Added initial files for pypi publish

### PR DESCRIPTION
**Pull request by baulus**

### CODE REVIEW

File: car_framework/car_service.py [hunk 1]
```diff
@@ -172,6 +173,78 @@ def get_db_status(self):
             raise UnrecoverableFailure('Getting the following status code when accessing ISC CAR service: %d' % status_code)
 
 
+    def graph_search(self, resource, search_id, source=""):
```
Comment (baulus) [inline car_framework/car_service.py:176]: Taken from: https://github.ibm.com/UDS/CAR-AWS-connector/blob/51f4d6d48ed385c1601e10adb3c270e5deec8cd7/base/car_service_client.py#L286

File: car_framework/car_service.py [hunk 2]
```diff
@@ -172,6 +173,78 @@ def get_db_status(self):
...
+
+        if r.status_code == 200:
+            return get_json(r)
+        elif recoverable_failure_status_code(r.status_code):
+            raise RecoverableFailure('Error occurred while searching collection: %d' % r.status_code)
+        else:
+            raise UnrecoverableFailure('Error occurred while searching collection: %d' % r.status_code)
+            
+
+    def graph_attribute_search(self, resource, attribute, search_id):
```
Comment (baulus) [inline car_framework/car_service.py:194]: Taken from: https://github.ibm.com/UDS/CAR-AWS-connector/blob/51f4d6d48ed385c1601e10adb3c270e5deec8cd7/base/car_service_client.py#L312

File: car_framework/car_service.py [hunk 3]
```diff
@@ -172,6 +173,78 @@ def get_db_status(self):
...
+        r = self.communicator.get(url)
+        if r.status_code == 200:
+            return get_json(r)
+        elif recoverable_failure_status_code(r.status_code):
+            raise RecoverableFailure('Error occurred while searching collection attribute: %d' % r.status_code)
+        else:
+            raise UnrecoverableFailure('Error occurred while searching collection attribute: %d' % r.status_code)
+
+
+    def database_patch_value(self, tags):
```
Comment (baulus) [inline car_framework/car_service.py:206]: Taken from: https://github.ibm.com/UDS/CAR-AWS-connector/blob/51f4d6d48ed385c1601e10adb3c270e5deec8cd7/base/car_service_client.py#L363

File: car_framework/car_service.py [hunk 4]
```diff
@@ -172,6 +173,78 @@ def get_db_status(self):
...
+
+        if r.status_code == 200:
+            return get_json(r)
+        elif recoverable_failure_status_code(r.status_code):
+            raise RecoverableFailure('Error occurred while searching collection attribute: %d' % r.status_code)
+        else:
+            raise UnrecoverableFailure('Error occurred while searching collection attribute: %d' % r.status_code)
+ 
+    
+    def edge_patch(self, source, edge_id, data):
```
Comment (baulus) [inline car_framework/car_service.py:230]: Taken from: https://github.ibm.com/UDS/CAR-AWS-connector/blob/51f4d6d48ed385c1601e10adb3c270e5deec8cd7/base/car_service_client.py#L140

**Comment by baulus** [comment]
@omkar-g  
do we need to document how to remove the submodule from the currently existing car connectors, for them to use this pypi module instead?

Just in case here are the steps:

- Delete the relevant line from the .gitmodules file.
- Delete the relevant section from .git/config.
- Run git rm --cached path_to_submodule (no trailing slash).
- Commit and delete the now untracked submodule files.
- Add car-connector-framework==<VERSION> in requirements.txt

**Review by omkar-g — APPROVED** [review]

**Pull request merged**